app/views/groups/edit.html.erb

```diff
@@ -89,14 +89,10 @@
 <% end %>
 </ul>
 
-<script>
-pdc = {
-  ...pdc,
-  deleteUserFromGroupUrl: '<%= delete_user_from_group_url(id: @group.id, uid: "uid-placeholder") %>',
-  formAuthenticityToken: '<%= form_authenticity_token %>',
-  groupId: '<%= @group.id %>',
-  addSubmitterUrl: '<%= add_submitter_url(id: @group.id, uid: "uid-placeholder") %>',
-  addAdminUrl: '<%= add_admin_url(id: @group.id, uid: "uid-placeholder") %>',
-  userPath: "<%= user_path('user-placeholder') %>"
-}
-</script>
+<input type="hidden" id="deleteUserFromGroupUrl" name="deleteUserFromGroupUrl" value="<%= delete_user_from_group_url(id: @group.id, uid: "uid-placeholder") %>" />
```

Comment:
This is the change that made the JS constantly work.  From what I could see the global variable sometimes had the correct information and sometimes did not.  Changing to a hidden input made the information consistently present.